clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusClusterProvider.groovy

```diff
@@ -184,6 +190,20 @@ class TitusClusterProvider implements ClusterProvider<TitusCluster>, ServerGroup
     serverGroup.placement.region = region
     serverGroup.scalingPolicies = serverGroupData.attributes.scalingPolicies
     serverGroup.targetGroups = serverGroupData.attributes.targetGroups
+
+    getTaggedImageAttributes(account, job).ifPresentOrElse(
+      { it ->
+        serverGroup.buildInfo.jenkins =
+          [
+            "name"  : it.jenkinsJob ?: "NA",
+            "number": it.buildNumber ?: "NA",
```

Comment:
Is the key name 'number' obvious enough here, is it better than just leaving as `buildNumber`?

Reply:
i just followed the same naming convention we have for AWS Server groups , might be easier for deck in this case.